integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/jira/issues/DS1194NestedQueryEmptyResultTest.java

```diff
@@ -1,73 +0,0 @@
-/*
- * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
- *
- * WSO2 Inc. licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-package org.wso2.ei.dataservice.integration.test.jira.issues;
-
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
-import org.testng.Assert;
-import org.testng.annotations.AfterClass;
-import org.testng.annotations.BeforeClass;
-import org.testng.annotations.Test;
-import org.wso2.ei.dataservice.integration.test.DSSIntegrationTest;
-import org.wso2.ei.dataservice.integration.test.odata.ODataTestUtils;
-
-import java.io.File;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-import static org.wso2.ei.dataservice.integration.test.odata.ODataTestUtils.sendGET;
-
-public class DS1194NestedQueryEmptyResultTest extends DSSIntegrationTest {
```

Comment:
Can you please explain why this testcase is removed?

Reply:
Moved the test case here
https://github.com/wso2/product-ei/pull/1311/files#diff-9413b9cd5ef7fcc560af2025732fe70cR94